after/queries/bash/matchup.scm

```diff
@@ -1,5 +1,29 @@
 (if_statement
   "if" @open.if
   "fi" @close.if) @scope.if
+
 (else_clause
   "else" @mid.if.1)
+
+(elif_clause
+  "elif" @mid.if.2)
+
+((word) @mid.if.3 (#eq? @mid.if.3 "break"))
+((word) @mid.if.4 (#eq? @mid.if.4 "continue"))
```

Comment:
break/continue are associated with while/for, no?

Reply:
Good catch, they are usually nested in if statements which is why I put them there initially, just pushed a fix placing them in for and while loops.